include/deal.II/grid/tria_accessor.h

```diff
@@ -1238,6 +1238,34 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   double diameter () const;
 
   /**
+   * Return a pair of Point and double corresponding to the center and
+   * the radius of a reasonably small enclosing ball of the object.
+   *
+   * The function implements Ritter's O(n) algorithm to get a reasonably
+   * small enclosing ball around the vertices of the object.
+   * The initial guess for the enclosing ball is taken to be the ball
+   * which has the largest diagonal of the object as one of it's diameters.
```

Comment:
i would just write `...which contains the largest diagonal of the object`. To me `one of its diameters` sounds strange.